sqlc.yaml

```diff
@@ -8,3 +8,4 @@ sql:
         package: "db"
         out: "db"
         sql_package: "pgx/v5"
+        emit_pointers_for_null_types: true
```

Comment:
This was necessary to get the mapping working. Nullable text fields are now generated as `*string` in the repo. Otherwise it uses specific Pg-types but couldn't get that to work properly.

Reply:
Fixed in c3142283c8ad7b63b1a5f80815999ed9e4ea373d